Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,51 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "https://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "https://anongit.freedesktop.org/git/gstreamer/qt-gstreamer.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "gst-plugins-base"
+  depends_on "qt5"
+
+  def install
+    args = std_cmake_args
+    args << "-DQT_VERSION=5"
+    args << "-DUSE_GST_PLUGIN_DIR=OFF"
+    args << "-DUSE_QT_PLUGIN_DIR=OFF"
+    args << "-DCMAKE_CXX_FLAGS=-I#{Formula["gstreamer"].opt_lib}/gstreamer-1.0/include"
+    system "cmake", *args
+
+    # QtGStreamer won't build on a case insensitive filesystem,
+    # because its "Memory" header is found in the header search path
+    # instead of the "memory" header of standard C++.
+    # Temporarily rename it as a workaround.
+    # https://bugzilla.gnome.org/show_bug.cgi?id=746760
+    # https://bugzilla.gnome.org/show_bug.cgi?id=733788
+    # https://bugzilla.gnome.org/show_bug.cgi?id=763201
```

Comment:
Everything builds aside from that one thing, though. I've emailed one of the developers, but it sure would be nice to have this in before Monday and remove the workaround later. :)